docs/user/openstack/README.md

```diff
@@ -322,9 +322,9 @@ If you have created the API DNS record, you should be able access the OpenShift
 
 ##### External Ingress (apps) Access
 
-The installer doesn't currently handle the Ingress floating IP address the same way it does the API one.
+In the same manner, you may have specified an Ingress floating IP by adding the `ingressFloatingIP` entry in your `install-config.yaml`, in which case the installer attaches the Floating IP address to the `ingress-port` automatically.
 
-To make the OpenShift Ingress access available (this includes logging into the deployed cluster), you will need to attach the Ingress floating IP to the `ingress-port` after the cluster is created.
+If not, you can always attach the Ingress floating IP to the `ingress-port` after the cluster is created.
```

Comment:
@mandre Would you say that providing ingressFloatingIP at installation is preferred to providing the FIP post-installation, now?

Reply:
I would say so, this makes it for a more streamlined installation flow if you add the Ingress FIP to `install-config.yaml` right from the start.

The setting is not exposed via the wizard, but it's not a big issue since you can still create the Ingress FIP post-install and attach it to the ingress port as before.